src/style.css

```diff
@@ -1 +1,31 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-image: url('./images/star.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
```

Comment:
Avoid fixing the size of containers like `.stars__star` if there is no specific requirement for it. Let the content size dictate the container size to avoid overflow or accidental scrollbars.